tests/torchtune/modules/test_vision_transformer.py

```diff
@@ -173,13 +176,15 @@ def test_vision_transformer_return_hidden_layers(self, transformer_config):
                 hidden_layer.shape == expected_shape_hidden_layers
             ), f"Expected shape {expected_shape_hidden_layers}, but got {hidden_layer.shape=}"
 
+        # Target based off of reference nn.TransformerEncoderLayer implementation
         assert_expected(
             torch.stack(hidden_layers, dim=-1).mean(),
-            torch.tensor(6.6938),
+            torch.tensor(8.3112),
```

Comment:
Wizadry!!

Reply:
I see now it's based on `nn.TransformerEncoderLayer`. Do you have a gist or something to point to by any chance?